tests/scripts/remove_ip.sh

```diff
@@ -2,9 +2,9 @@
 
 set -euo pipefail
 
-INTF_IDX_LIST=$(cat /proc/net/dev | grep eth | awk -F'eth|:' '{print $2}')
+INTF_LIST=$(ls /sys/class/net | grep -E "^eth[0-9]+$")
 
-for i in ${INTF_IDX_LIST}; do
-  echo "Flush eth${i} IP address"
-  ip address flush dev eth$i
+for INTF in ${INTF_LIST}; do
+    echo "Flush ${INTF} IP address"
+    ip addr flush dev ${INTF}
 done
```

Comment:
Second remove ip?

Reply:
one file and reference to a file from both places?

Reply:
Yes. When reference it from scripts under sonic-mgmt/tests, it is like:

sonic-mgmt/tests/fdb/test_fdb.py ---call---> sonic-mgmt/tests/scripts/remove_ip.sh ---symbolic link of---> sonic-mgmt/ansible/roles/test/files/helpers/remove_ip.sh